packages/react-dom/src/__tests__/ReactBrowserEventEmitter-test.internal.js

```diff
@@ -66,7 +65,8 @@ describe('ReactBrowserEventEmitter', () => {
     React = require('react');
     ReactDOM = require('react-dom');
     ReactDOMComponentTree = require('../client/ReactDOMComponentTree');
-    DOMEventPluginSystem = require('../events/DOMEventPluginSystem');
+    listenToEvent = require('../events/DOMLegacyEventPluginSystem')
```

Comment:
Could you keep this named as legacyListenToEvent? 
```suggestion
    legacyListenToEvent = require('../events/DOMLegacyEventPluginSystem')
```

Reply:
@threepointone The `ReactBrowserEventEmitter-test.internal.js` change is because I fork that file in the follow up that uses the flag. Also, I'd rather add the flag in this PR as it feels more churny having another PR that adds it but doesn't use it (lots of file changes makes the diff harder to review).